src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -138,7 +138,14 @@ internal void AddProvider(string workerDir)
                         if (workerDescriptionProfiles.Count > 0)
                         {
                             _profileManager.SetWorkerDescriptionProfiles(workerDescriptionProfiles, workerDescription.Language);
-                            _profileManager.LoadWorkerDescriptionFromProfiles(workerDescription, out workerDescription);
+                            var profileLoaded = _profileManager.LoadWorkerDescriptionFromProfiles(workerDescription, out workerDescription);
+
+                            // Skip the worker if SkipWhenProfileConditionsUnmet is true and none of the profiles were loaded.
+                            if (workerDescription.SkipWhenProfileConditionsUnmet && !profileLoaded)
+                            {
+                                _logger.LogInformation("Skipping WorkerConfig for language: {workerDescription.Language} since none of the profiles loaded and 'SkipWhenProfileConditionsUnmet' is true.", workerDescription.Language);
```

Comment:
Is `{workerDescription.Language}` here intentional instead of `{language}` or `{workerLanguage}`?

Can we also change the terminology here from language to stack?

Reply:
fixed in the new log message.